source/usr/lib/mcvirt/auth/user.py

```diff
@@ -18,14 +18,21 @@
 # along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
 
 from mcvirt.auth.user_base import UserBase
-from mcvirt.exceptions import OldPasswordIncorrect
+from mcvirt.auth.permissions import PERMISSIONS
+
+import Pyro4
```

Comment:
We normally import external modules before internal ones, with a single line deliiter, e.g.

    # End of license
    
    import Pyro4
    
    from mcvirt.blah import BiggerBlah
    
    
    class TestMe(object)...